src/lib.rs

```diff
@@ -172,7 +172,7 @@ impl ToRepository for mongodb::Database {
 pub mod prelude {
     #[doc(no_inline)]
     pub use crate::mongo::bson::{
-        bson, de::Error as BsonDeError, doc, from_bson, oid::ObjectId, ser::Error as BsonSerError,
```

Comment:
This is a breaking change, next version should be `0.8.0`.

Reply:
Wait, I didn't want to remove anything there, this is a mistake.
I will revert this.